src/components/application_manager/src/commands/hmi/sdl_activate_app_request.cc

```diff
@@ -55,9 +55,15 @@ void SDLActivateAppRequest::Run() {
       application_manager_.application(application_id);
 
   if (!app) {
```

Comment:
@AGritsevich is there a reason to try to find `application` in common way above?